docker/Dockerfile

```diff
@@ -102,7 +102,9 @@ FROM ubuntu:bionic as common_cpp_image
 RUN apt-get update \
  && apt-get install -y -q \
     pkg-config \
+    wget \
     cmake \
+    libprotobuf-dev \
```

Comment:
We dont need protobuf. please remove

Reply:
It is required otherwise below dpkg complain while installing packages.

Reply:
Ok